pub/errors/processor.php

```diff
@@ -265,7 +265,8 @@ public function getHostUrl()
             $host = 'localhost';
         }
 
-        $isSecure = (!empty($_SERVER['HTTPS'])) && ($_SERVER['HTTPS'] != 'off');
+        // HTTP_X_FORWARDED_PROTO to check whether a webserver using HTTP is behind a load balancer serving HTTPS
+        $isSecure = (!empty($_SERVER['HTTPS'])) && ($_SERVER['HTTPS'] != 'off') || $_SERVER['HTTP_X_FORWARDED_PROTO'] === 'https';
```

Comment:
Please add checking that `$_SERVER['HTTP_X_FORWARDED_PROTO']` exists, otherwise we'll have notice there